pkg/asset/quota/quota.go

```diff
@@ -118,7 +122,23 @@ func (a *PlatformQuotaCheck) Generate(dependencies asset.Parents) error {
 			return summarizeFailingReport(reports)
 		}
 		summarizeReport(reports)
-	case azure.Name, baremetal.Name, libvirt.Name, none.Name, openstack.Name, ovirt.Name, vsphere.Name:
+	case typesopenstack.Name:
+		opts := &openstackclientconfig.ClientOpts{Cloud: ic.Config.Platform.OpenStack.Cloud}
+		os.Unsetenv("OS_CLOUD")
+		q, err := quotaopenstack.Load(opts)
```

Comment:
Hmm, we probably want to store the quota info in the `CloudInfo` stuct.

Then we could call `GetCloudInfo` here. This would take care of unsetting the OS_CLOUD variable for us and make it easier to deal with self-signed certificates.

Reply:
ack, looking at it now.